.github/workflows/ci.yml

```diff
@@ -1053,6 +1087,40 @@ jobs:
         export
         echo $GITHUB_ENV
 
+    - name: Rebuild CppInterOp in Debug mode for tmate session
+      if: ${{ runner.os != 'windows' && (matrix.coverage == true) &&  failure() }}
+      run: |
+        cling_on=$(echo "${{ matrix.cling }}" | tr '[:lower:]' '[:upper:]')
+        
+        # Build CppInterOp next to cling and llvm-project.
+        cd build
+        rm -rf ./*
+        cling_on=$(echo "${{ matrix.cling }}" | tr '[:lower:]' '[:upper:]')
```

Comment:
we can drop this rebuild

Reply:
You mentioned that if you have to ssh into the machine it will need to be rebuilt with debug. This saves time by getting the ci to rebuild, and also guards again the case the person sshing into the macine forgets to (we know too, but others may not).

Reply:
@maximusron I can revert this change if your sure but I think its a good guard against those sshing in..

Reply:
> We can make it conditional on if the runner is run in debug.

@vgvassilev Currently we have no debug flag. I can make it dependent on a debug run when I address https://github.com/compiler-research/CppInterOp/issues/262 soon if that's ok?